tikv-client/src/main/java/com/pingcap/tikv/TiConfiguration.java

```diff
@@ -32,27 +28,120 @@
 import lombok.Setter;
 import lombok.experimental.Accessors;
 import org.joda.time.DateTimeZone;
+import org.tikv.common.HostMapping;
+import org.tikv.common.Utils;
+import org.tikv.common.pd.PDUtils;
+import org.tikv.common.region.TiStoreType;
+import org.tikv.common.util.BackOffer;
 import org.tikv.kvproto.Kvrpcpb.CommandPri;
 import org.tikv.kvproto.Kvrpcpb.IsolationLevel;
+import org.tikv.shade.com.google.common.collect.ImmutableList;
 
 @Getter
 @Setter
 @Accessors(chain = true)
 public class TiConfiguration implements Serializable {
-  private static final DateTimeZone DEF_TIMEZONE = Converter.getLocalTimezone();
+
+  // --------------- different ------------------
+  // |item          | tispark    | client-java  |
+  // --------------------------------------------
+  // |timeout       | 10minutes  | 200ms        |
+  // |maxFrameSize  | 2Gb        | 512MB        |
+  // --------------------------------------------
+  private int maxFrameSize = DEF_MAX_FRAME_SIZE;
+  private static final int DEF_MAX_FRAME_SIZE = 2147483647; // 2 GB
+  private int timeout = DEF_TIMEOUT;
+  private TimeUnit timeoutUnit = DEF_TIMEOUT_UNIT;
   private static final int DEF_TIMEOUT = 10;
   private static final TimeUnit DEF_TIMEOUT_UNIT = TimeUnit.MINUTES;
-  private static final int DEF_SCAN_BATCH_SIZE = 10480;
-  private static final boolean DEF_IGNORE_TRUNCATE = true;
+
+  // --------------   only in tispark -------------
```

Comment:
Can those configs only in tispark's configs still work ?

Reply:
yes